fb.c

```diff
@@ -161,12 +161,13 @@ static int vcam_fb_check_var(struct fb_var_screeninfo *var,
     var->xres_virtual = var->xoffset + var->xres;
     var->yres_virtual = var->yoffset + var->yres;
 
+    /* check bpp */
     if (var->bits_per_pixel <= 16)
-        var->bits_per_pixel = 16;
+       var->bits_per_pixel =  ALIGN(16, 4);
```

Comment:
No, you can avoid the repeated if-else statements by `ALIGN` macro.

Reply:
the bpp = 24 is not the power of two.
I thought that write on below.
```cpp
int bpp = var->bits_per_pixel;
if (bpp > 16 && bpp <= 24)
    var->bits_per_pixel = 24;
else
    var->bits_per_pixel = ALIGN(bpp, 4);
```